frontend/packages/console-plugin-sdk/src/registry.ts

```diff
@@ -136,10 +137,14 @@ export class ExtensionRegistry {
     return this.extensions.filter(isClusterServiceVersionAction);
   }
 
-  public getKebabActions() {
+  public getPersistentVolumeClaimActionActions() {
     return this.extensions.filter(isKebabActions);
   }
 
+  public getOCSKebabActions() {
+    return this.extensions.filter(isPersistentVolumeClaimAction);
+  }
+
```

Comment:
Why we changed name for `getKebabActions`?

Reply:
In `getPersistentVolumeClaimActionActions` we are trying to filter `isKebabActions`?

Reply:
By mistake, I renamed the wrong function name. It should be for `getOCSKebabActions`